docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -1,5 +1,24 @@
 [[synthetics-private-location]]
-= Add a private location
+= Monitor a private endpoint
+
+To monitor resources on private networks use {{agent}} to create a Private Location, or allow access to your private endpoints to the Synthetics Service.
```

Comment:
A couple thoughts:

* I would order the options the same way they are organized on the page. It looks like you put the private endpoint section before creating a private location.
* This sentence was confusing to me:

  >allow access to your private endpoints to the Synthetics Service

  but feel free to adjust my suggestion changed the meaning.

```suggestion
To monitor resources on private networks you can either:

* Allow the Synthetics service to access your private endpoints.
* Use {agent} to create a Private Location.

```

Reply:
Much cleaner, yes :)